File: Python Code/Daily_Flash_Code_0/Week9/21Mar/53-DailyFlash_Solutions/21_March_Solution_Three/C++/prog1.cpp
#include<iostream>
#include<math.h>
int main() {
	int l, mul = 0;
	std::cin >> l;
	for(int i = 1; i <= l; i++) {
		mul += pow(i, i) * (i - 1);
	}
	std::cout << mul;
}

File: Python Code/Daily_Flash_Code_0/Week9/21Mar/53-DailyFlash_Solutions/21_March_Solution_Three/Java/prog1.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		int l, mul = 0;
		Scanner sc = new Scanner(System.in);
		l = sc.nextInt();
		for(int i = 1; i <= l; i++) {
			mul += Math.pow(i, i) * (i - 1);
		}
		System.out.println(mul);
	}
}
